### PR TITLE
main/mesa: upgrade to 19.0.6

### PR DESCRIPTION
**Pull request by prspkt**
~~Depends on #7259~~

### CODE REVIEW

File: main/mesa/add-glx-use-tls.patch [hunk 1]
```diff
@@ -0,0 +1,47 @@
+From f895a39b7437381a86383dd97250aa2cc7393bff Mon Sep 17 00:00:00 2001
+From: maxice8 <thinkabit.ukim@gmail.com>
```
Comment (maxice8) [inline main/mesa/add-glx-use-tls.patch]: nice

**Comment by maxice8** [comment]
https://github.com/alpinelinux/aports/pull/7259 This is required, libdrm>=2.4.97

**Comment by tanertas** [comment]
Please check this bug report regarding enabling glx-tls.
https://bugs.alpinelinux.org/issues/9260

**Comment by maxice8** [comment]
rebase to get a libdrm version that can be used to build it.

**Comment by maxice8** [comment]
:crab: :crab: MERGE THIS :crab: :crab:

**Comment by maxice8** [comment]
19.0.6 is the correct version

From the release notes

>  Mesa 19.1.0 is a new development release. People who are concerned with stability and reliability should stick with a previous release or wait for Mesa 19.1.1.

**Comment by prspkt** [comment]
It might be a good idea to merge this after making [llvm8](https://github.com/alpinelinux/aports/pull/8620/commits) the default provider.

**Pull request closed without merging**